integrations/TVSquared/browser.js

```diff
@@ -0,0 +1,104 @@
+import ScriptLoader from "../ScriptLoader";
+
+class TVSquared {
+  constructor(config) {
+    this.brandId = config.brandId;
+    this.clientId = config.clientId;
+    this.eventWhiteList = config.eventWhiteList || [];
+    this.customMetrics = config.customMetrics || [];
+    this.name = "TVSquared";
+  }
+
+  init() {
+    logger.debug("===in init TVSquared===");
+    window._tvq = window._tvq = [];
+    var url = document.location.protocol == "https:" ? "https://" : "http://";
+    url += "collector-" + this.clientId + ".tvsquared.com/";
+    window._tvq.push(["setSiteId", this.brandId]);
+    window._tvq.push(["setTrackerUrl", url + "tv2track.php"]);
+    ScriptLoader("TVSquared-integration", url + "tv2track.js");
+    window._tvq.push([
+      function () {
+        this.deleteCustomVariable(5, "page");
+      },
+    ]);
+  }
+
+  isLoaded() {
+    logger.debug("in TVSqaured isLoaded");
+    return !!window_tvq;
```

Comment:
typo? should be !!window._tvq

should we not do something like - 
return !!(window._tvq && window._tvq.push !== Array.prototype.push);

Reply:
yes it was a type changed it.